x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_about_rule/default_value.ts

```diff
@@ -6,6 +6,14 @@
 
 import { AboutStepRule } from '../../types';
 
+export const threatsDefault = [
+  {
+    framework: 'MITRE ATT&CK',
```

Comment:
Does `framework` find itself user facing? If so, this'll need `i18n`'d.

There's one here as well:

https://github.com/elastic/kibana/blob/fa8f61c3666f29f289c8b98e33b03755e19a4241/x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/create/helpers.ts#L94

Reply:
it is not user facing, just for API